gensim/models/wrappers/fasttext.py

```diff
@@ -284,12 +285,12 @@ def load_model_params(self, file_handle):
     def load_dict(self, file_handle, encoding='utf8'):
         vocab_size, nwords, _ = self.struct_unpack(file_handle, '@3i')
         # Vocab stored by [Dictionary::save](https://github.com/facebookresearch/fastText/blob/master/src/dictionary.cc)
-        assert len(self.wv.vocab) == nwords, 'mismatch between vocab sizes'
-        assert len(self.wv.vocab) == vocab_size, 'mismatch between vocab sizes'
+        logger.info("loading projection weights")
```

Comment:
Yes, but here we aren't loading exactly the same thing as in the case linked above - in that case, we're loading the vocab words along with their vectors, here we're simply building a dictionary.